app/src/interfaces/_system/folder/folder.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+	<folder-picker :value="currentFolder" :disabled-folders="disabledFolders" @input="onFolderSelect" />
```

Comment:
Instead of having a duplicate local state, you can use the `value` prop here directly

```suggestion
	<folder-picker :value="value" :disabled-folders="disabledFolders" @input="onFolderSelect" />
```